sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 1.41.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.41.0 (2024-07-11)
 
 ### Bugs Fixed
-- Fixed an issue that could result in `BearerTokenAuthenticationPolicy` fails to refresh a token, resulting in a `OperationCanceledException`.
-
-- Fixed case where a GeoJSON string could not be deserialized when the BoundingBox JSON value ("bbox") was set explicitly to null ([#44835](https://github.com/Azure/azure-sdk-for-net/pull/44835))
 
-### Other Changes
+- Fixed an issue that could result in `BearerTokenAuthenticationPolicy` fails to refresh a token, resulting in a `OperationCanceledException` ([#44882](https://github.com/Azure/azure-sdk-for-net/pull/44882)).
```

Comment:
```suggestion

- Fixed an issue that could result in `BearerTokenAuthenticationPolicy` fails to refresh a token, resulting in a `OperationCanceledException` ([#44882](https://github.com/Azure/azure-sdk-for-net/pull/44882)).
```

Reply:
Per offline discussion, necessary line break is present; misperceived due to a wonky diff.